### PR TITLE
Support -v, -vv, -vvv, -q, -qq, -qqq for verbosity

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

|short option|long option|
|-|-|
|-qqq|--verbosity=quiet|
|-qq|--verbosity=error|
|-q|--verbosity=warning|
||--verbosity=info (default)|
|-v|--verbosity=verbose|
|-vv|--verbosity=debug|
|-vvv|--verbosity=trace|

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Try it!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,11 @@ Every entry has a category for which we use the following visual abbreviations:
 - ⚡️ breaking change
 - 🐞 bugfix
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- 🎁 The new short options `-v`, `-vv`, `-vvv`, `-q`, `-qq`, `-qqq` map onto the
+  existing verbosity levels. This aligns VAST's command line interface with the
+  user expectations. [#1244](https://github.com/tenzir/vast/1244)
```
Comment (mavam) [inline CHANGELOG.md:17]:
```suggestion
  existing verbosity levels. 
  [#1244](https://github.com/tenzir/vast/1244)
```

**Review by mavam — COMMENTED** [review]
I tested this as follows:

```
vast start
vast -v start
vast -vv start
vast -vvv start
```

All good. Then I tried the "quiet" mode by trying to bind twice to the same port with two instances of `vast start`.

```
vast -q start
!! filesystem_error: "PID file found: " "[..]/vast.db/pid.lock" "terminate process" "95957"
```

I get this for all variations of quietness, i.e., `-q`, `-qq`, and `-qqq`. Is this expected behavior?

**Comment by dominiklohmann** [comment]
> I get this for all variations of quietness, i.e., `-q`, `-qq`, and `-qqq`. Is this expected behavior?

Yes, because that error is printed on stderr, and without going through any of our logger macros.

**Review by mavam — APPROVED** [review]
Nice work. Small but really handy feature!

**Pull request merged**